style-guide/index.md

```diff
@@ -227,48 +227,49 @@ title: "Canada.ca Content Style Guide"
     <ul>
       <li>guide your audience to the information they need to complete their task </li>
       <li>place supporting and relevant details toward the end of the text </li>
     </ul>
-    <figure class="mrgn-bttm-lg">
-      <figcaption class="text-center"> <strong>Figure 1</strong> </figcaption>
-      <img alt="Pyramid upside down showing how to start a web page with key information first. Text version below:" class="img-responsive center-block" id="fig1" src="/style-guide/images/fig-01-eng.jpg">
-      <details class="brdr-tp brdr-rght brdr-bttm brdr-lft">
-        <summary class="wb-toggle" data-toggle="'{&quot;print&quot;:&quot;on&quot;}'"> Figure 1 - Text version </summary>
-        <p>Inverted pyramid divided into four sections, from top to bottom. The image shows the order of information on a web page.</p>
+    <div class="row mrgn-tp-lg mrgn-bttm-md">
+      <div class="col-md-8">
+        <figure><img alt="Upside-down pyramid showing how to start a web page with key information first." class="img-responsive" id="fig1" src="/style-guide/images/pyramid-en.png">
+          <figcaption>
+            <p><strong>Figure 1:</strong> inverted pyramid for ordering content on a web page</p>
+          </figcaption>
+        </figure>
+      </div>
+    </div>
+    <p>The inverted pyramid for ordering content on a web page in Figure 1 is divided into four sections.  The most important content for task completion is at the top, the widest part of the pyramid.   The relevance of content for task completion flows down the inverted pyramid toward the peak at the bottom.</p>
+    <ol>
+      <li><strong>Most important</strong>:
         <ul>
-          <li>First:
-            <ul>
-              <li>start with the most important idea, step or information</li>
-            </ul>
-          </li>
-          <li>Second:
-            <ul>
-              <li>add details and tasks in order of importance</li>
-              <li>keep all task related content in the centre of the page</li>
-              <li>use links to direct people to key information and tasks</li>
-            </ul>
-          </li>
-          <li>Third:
-            <ul>
-              <li>include information for audiences who want to learn more or have more knowledge of the topic or service, but only if it makes sense</li>
-            </ul>
-          </li>
-          <li>Fourth:
-            <ul>
-              <li>add links only if they relate to content that supports completion of the task on the web page (for example, laws or publications)</li>
-            </ul>
-          </li>
+          <li>start with the most important idea, step or information</li>
         </ul>
-      </details>
-    </figure>
+      </li>
+      <li><strong>Prioritized tasks</strong>:
+        <ul>
+          <li>add details and tasks in order of importance</li>
+          <li>keep all task related content in the centre of the page</li>
```

Comment:
please add a hyphen between task and related, so: 'task-related content'

Reply:
Done